light-client/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "light-client"
+version = "0.1.0"
+authors = ["Romain Ruetschi <romain@informal.systems>"]
+edition = "2018"
+
+[dependencies]
+tendermint = { path = "../tendermint" }
+
+anomaly = { version = "0.2.0", features = ["serializer"] }
+derive_more = "0.99.5"
+serde = "1.0.106"
+serde_derive = "1.0.106"
+thiserror = "1.0.15"
+futures = "0.3.4"
+tokio = "0.2.20"
+prost-amino = "0.5.0"
+contracts = "0.4.0"
+sled = "0.31.0"
+serde_cbor = "0.11.1"
```

Comment:
Yet another binary encoding 😱 😄 I assume this is the most reasonable choice to use in combination with sled?

Reply:
I have to admit I didn't give it too much thought. I initially considered just serializing keys and values to JSON and then to bytes, but that seemed a bit wasteful, so I figured using a dedicated binary encoding was better. In the end, aside from the extra dependencies, the choice of encoding does not matter since it is internal to the light store and specific to the choice of `sled` as a database. But I'd be happy to discuss alternatives, either for the binary encoding, or for `sled` :)

Reply:
Not a big deal but couldnt prost work here for proto3?